.eslintrc.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
   extends: '@mate-academy/eslint-config',
   env: {
-    jest: true
+    jest: true,
   },
   rules: {
-    'no-proto': 0
+    'no-proto': 0,
```

Comment:
The 'no-proto' rule is set to 0, which disables it. If this is not intentional, consider setting it to 'warn' or 'error' to enforce the rule.